spelling/action.yml

```diff
@@ -7,6 +7,10 @@ inputs:
   GITHUB_TOKEN:
     description: 'Github secret Token'
     required: true
+  FILEPATHS:
+    description: 'Filepaths to look at, specified as a comma separated string ie "docs,blog,etc"'
+    required: false
+    default: 'docs'  
```

Comment:
let's test with this, the issue is MM which doesn't have doc only in docs at root